src/main/kotlin/racingcar/view/ResultView.kt

```diff
@@ -0,0 +1,30 @@
+package racingcar.view
+
+import racingcar.domain.Car
+
+class ResultView {
+    fun printRacingResult() {
+        println("실행 결과")
+    }
+
+    fun printCurrentRace(cars: List<Car>) {
+        for (car in cars) {
+            println(mapForDisplay(car.distance))
+        }
```

Comment:
foreach 를 사용해보는 것은 어떨까요?
```suggestion
        cars.forEach { println(mapForDisplay(it.distance)) }
```

Reply:
👍 
컬렉션에서 루프 동작만 수행하고 종료시키는 경우 stream 보다 기본적인 for문 사용을 선호했는데, 출력의 경우엔 forEach를 사용하는 것도 좋은거 같습니다!